src/drivers/boards/common/board_dma_alloc.h

```diff
@@ -112,7 +114,7 @@ __EXPORT int board_get_dma_usage(uint16_t *dma_total, uint16_t *dma_used, uint16
 #if defined(BOARD_DMA_ALLOC_POOL_SIZE)
```

Comment:
@dagar Pardon my noobness, I want to make sure I understand what's going on. So the way I understand this is: by providing this definition *BOARD_DMA_ALLOC_POOL_SIZE* we are telling the linker about the `board_dma_*` functions (that is what the __EXPORT does) which makes the functions available for use as long as we `#include <drivers/boards/common/board_dma_alloc.h>`?

Reply:
Also, is `board_dma_alloc.cpp` something you wrote?

Reply:
Yes, and it's just a slight generalization of the existing FAT DMA allocator.